### ISSUE TITLE
Form embeds

### ISSUE THREAD

**Issue by lucastbelem** [themeetinghouse/mobile#162]
Is it possible to embed a formstack in an app (like we did for the website) but have it underneath native design?

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=603867006)

**Comment by GeorgeBellTMH** [comment]
It may be possible, but we'd have to try. There is a way to essentially embed a web browser and load content in it. We need to decide if that makes sense given the long term plan and goals. It may be better to design form components that we can reuse and create a more integrated experience.

**Comment by lucastbelem** [comment]
@GeorgeBellTMH yeah, that's true. If we went that route though, I'd want to do something that can be applied to the app and the website concurrently. Then we move away from formstack and use our own thing for everything.
